### ISSUE TITLE
CUDA Toolkit 6.0?

### ISSUE THREAD

**Issue by MikeyCarter** [cbuchner1/CudaMiner#142]
Any news on supporting the latest toolkit?

**Comment by jsmith-uk** [comment]
CUDA Toolkit 6.0 has dropped support for devices with compute capability below 2.0. Get rid of -arch=compute_10 in Makefile.am, and everything will compile just fine.

**Comment by StaleHyena** [comment]
Yeah, but when i do replace compute_10 by sm_21 i get a bad .bss section on the linker... I haven't tried using the intended cuda version though, so better do that before calling it quits
